docs/pages/setup/reference/cli.mdx

```diff
@@ -1289,6 +1289,68 @@ Print cluster version:
 tctl version
 ```
 
+## Resource Filtering
+
+### tsh and tctl ls
+
+Resources nodes, apps, db, and kube all use the same filter flags and argument.
+
+#### Argument
+
+`[<labels>]` List of comma separated labels to filter by labels (e.g. key1=value1,key2=value2)
```

Comment:
I think there's too many headings here. Can we simplify? Maybe something like this:

```suggestion
Both `tsh` and `tctl` allow you to filter node, app, db, and kube resources using the `--search` and `--query` flags.

The `--search` flag performs a simple fuzzy search on resource fields. For example, `--search=mac` searches for resources containing `mac`.

The `--query` flag allows you to perform more sophisticated searches using a predicate language that is detailed in the next section.

In both cases, you can further refine the results by appending a list of comma-separated labels to the command. For example:

```
$ tsh ls --search=term foo=bar
```
```